include/deal.II/matrix_free/evaluation_template_factory_hanging_nodes.templates.h

```diff
@@ -23,7 +23,7 @@
 #include <deal.II/matrix_free/evaluation_kernels_hanging_nodes.h>
 #include <deal.II/matrix_free/evaluation_template_factory.h>
 #include <deal.II/matrix_free/evaluation_template_factory_internal.h>
-#include <deal.II/matrix_free/fe_evaluation_base_data.h>
+#include <deal.II/matrix_free/fe_evaluation.h>
```

Comment:
Why do you need this file, which in turn includes a larger part of the library? I would only expect that you need to include `shape_info.h`?

Reply:
I guess I accidentally reverted the change from your PR that was merged yesterday^^